docs/source/conf.py

```diff
@@ -56,18 +58,13 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = "sphinx_rtd_theme"
+html_theme = "conda_sphinx_theme"
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
-html_style = "css/custom.css"
-
-# The name of an image file (relative to this directory) to use as a favicon of
-# the docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
-# pixels large.
-html_favicon = "img/conda-logo.png"
+# html_style = "css/custom.css"
```

Comment:
Oh, it's removing the custom css?

Reply:
I think we want to include the custom CSS file for some things. Will re-enable

Reply:
thanks!